examples/layer-browser/src/examples/core-layers.js

```diff
@@ -117,11 +117,17 @@ const PolygonLayerExample = {
     getFillColor: f => [200 + Math.random() * 55, 0, 0],
     getLineColor: f => [0, 0, 0, 255],
     getWidth: f => 20,
-    getHeight: f => Math.random() * 1000,
+    getElevation: f => Math.random() * 1000,
     opacity: 0.8,
     pickable: true,
-    lightSettings: LIGHT_SETTINGS
-  }
+    lightSettings: LIGHT_SETTINGS,
+    elevationScale: 0.6
+  },
+  getUpdateTriggers: settings => ({
```

Comment:
Is this intended to create a manual test case for the `getElevation` updateTrigger?

Maybe we should add a separate button to trigger updateTriggers rather than hooking it to some arbitrary setting? 

The prop handling in the layer browser is already very hacky. I would like us to at least think twice before adding more hacks.

Reply:
I agree this is hacky at a glance. At the moment, our layer browser doesn't provide any mechanism to allow tests on `updateTriggers`. It really increases the risks that `updateTriggers` are not handled correctly in layers. Though I agree with you this probably should not be included in this PR, will remove it and leave it for further discussion